depends/README.rst

```diff
@@ -1,4 +1,26 @@
 Depends
 =======
 
-Scripts in this directory can be used to download, build & install non-packaged dependencies; useful for testing with Travis CI.
+``install_openjpeg.sh`` and ``install_webp.sh`` can be used to
+download, build & install non-packaged dependencies; useful for
+testing with Travis CI.
+
+The other scripts can be used to install all of the dependencies for
+the listed operating systems/distros. The ``ubuntu_14.04.sh`` and
+``debian_8.2.sh`` scripts have been tested on bare AWS images and will
+install all required dependencies for the system python 2.7 and 3.4
+for all of the optional dependencies.  Git may also be required prior
+to running the script to actually download Pillow.
+
+e.g.::
+
+  $ sudo apt-get install git
+  $ git clone https://github.com/python-pillow/Pillow.git
```

Comment:
This isn't a recipe, it's an example of what else is required.  Though, a checkout in there probably wouldn't hurt. 

(realizing, sadly, that anything here is going to wind up as copy/paste)